docs/content/docs/deployment/filesystems/gcs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Google Cloud Storage
+weight: 3
+type: docs
+aliases:
+  - /deployment/filesystems/gcs.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Google Cloud Storage
+
+[Google Cloud storage](https://cloud.google.com/storage) (GCS) provides cloud storage for a variety of use cases. You can use it for **reading** and **writing data**.
+
+
+
+You can use GCS objects like regular files by specifying paths in the following format:
+
+```plain
+gs://<your-bucket>/<endpoint>
+```
+
+The endpoint can either be a single file or a directory, for example:
+
+```java
+// Read from GSC bucket
+env.readTextFile("gs://<bucket>/<endpoint>");
+
+// Write to GCS bucket
+stream.writeAsText("gs://<bucket>/<endpoint>");
+
+```
+
+### Libraries
+
+You must include the following libraries in Flink's `lib` directory to connect Flink with gcs:
```

Comment:
Because the packages below are described in maven format, let's maybe make it clear that we are talking about copying JARs into the lib folder.

Reply:
Using the keyword `jars` instead of `libraries` in the first line.
`You must include the following jars in Flink's lib`. I hope that makes it more clear